.github/workflows/gradle.yml

```
@@ -6,7 +6,7 @@ on:
   push:
     branches: [ "main", "SRU2024_v10" ]
   pull_request:
-    branches: [ "main" ]
+    branches: [ "main", "SRU2024_v10" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)